src/dev/bazel/jsts_transpiler.bzl

```diff
@@ -2,7 +2,7 @@
 
 load("@npm//@babel/cli:index.bzl", _babel = "babel")
 
-def jsts_transpiler(name, srcs, build_pkg_name, web = False, root_input_dir = "src", additional_args = ["--quiet"], **kwargs):
+def jsts_transpiler(name, srcs, build_pkg_name, web = False, root_input_dir = "", additional_args = ["--quiet"], **kwargs):
```

Comment:
Why can't we just get rid of the `root_input_dir` completely and add move the `index.ts` file in packages to the root? I feel like this change allows some packages to have a "root src dir" and other to not, and requires configuration that will end up in the `kibana.jsonc` file which really shouldn't need to be there.

Reply:
We've synced locally and we agreed we will use this PR as a first step before removing the src folder (and the support for it on the macro) completely in a subsequent PR that will update every package we currently have. The changes on that subsequent PR, other than removing the src folder itself, would be to also make sure we have an index.{js,ts} at the root of the package.

Reply:
Oh, apologies, I should have finished my review 👍